api/go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/nmstate/kubernetes-nmstate/api
 go 1.17
 
 require (
-	github.com/onsi/ginkgo v1.16.5
+	github.com/onsi/ginkgo/v2 v2.1.3
```

Comment:
maybe worth to add replace block at the end, so it will pin the version?
same on the ./go.mod

Reply:
The `sed` command did the job `replace` not needed.

Reply:
yes indeed, it is just for next time, so no surprise change happens, but whatever u prefer

Reply:
Prefer having it explicit is very misleading to see an "import" that does not have the stuff from the source file.